### ISSUE TITLE
No highlight for critic syntax in "Markdown + Critic"

### ISSUE THREAD

**Issue by Kabouik** [CriticMarkup/CriticMarkup-toolkit#29]
I know there was a commit a few months ago about "fixing highlight syntax", but I was not using Sublime Text 3 at this time so I cannot say if it was working at this time. Here, critic syntax is not highlighted with the `Markdown + Critic` language, whatever the color scheme used (`Critic` or `Monokai Soda + Critic`). 

Critic syntax highlight does appear when I select the `Critic` language from the bottom right list, but this language lacks all Markdown highlights and is very simple, and not as good as `Markdown + Critic` (at least in my opinion).

What should I do to enable critic syntax highlights with `Markdown + Critic` language? I have checked the _.tmLanguage_ files, and the content of `Critic.tmLanguage` is included in `Markdown + Critic.tmLanguage`, just it seems to be ineffective.

Thanks for your help.

**Comment by mberasategi** [comment]
Same problem here, no syntax highlighting whatever combination of language + specific color scheme for the language. Seeing that this is still "open" being so old, I am becoming somewhat pessimistic...

**Comment by Lywx** [comment]
@mberasategi You know that is a common issue for open source software. Just move on with your life if can't fix yourself : (

**Comment by mberasategi** [comment]
Yes, that's what I'm trying. I'm no programmer though, so I may not be able to. However, in case it helps, syntax highlighting is working with `Critic` language, but not with `Markdown + Critic`. Which makes it all even more strange to me...

**Comment by Lywx** [comment]
@mberasategi After some experiment, I barely get it working. I think all you should do is:

1. Copy every files in 

`\Sublime Text 3\Data\Packages\Critic Markup\Critic.*

to rename as 

`\Sublime Text 3\Data\Packages\Critic Markup\Markdown + Critic.* 

for every files for which there is no corresponding file matched to files for Critic syntax.

2. Copy XML node under Critic.tmLanguage:

```
...
<dict>

    ...
    <key>patterns</key>
    <array>

    [Everything in Here]

    </array>
</dict>
```

paste (insert) into the same node in newly copied and renamed "Markdown + Critic.tmLanguage" file.

3. Find uuid in "Markdown + Critic.tmLanguage", copy the uuid string into "Markdown + Critic.tmLanguage.JSON-tmLanguage"

Then the syntax highlighting should work. But the look is not very satisfactory for somehow bad configuration in some unknown place.

**Comment by mberasategi** [comment]
@Lywx I started down that row but felt that things were complicating a bit too much (that was before you posted). I discovered that **AcademicMarkdown supports Critic Markup syntax highlights** etc., so even if it's another thing to install, that's what I did. I modified the default AcademicMarkdown color scheme to make it [OcenaicNext](https://github.com/voronianski/oceanic-next-color-scheme/blob/master/screenshots/jsx-component.png)-dark (which I wanted) and am also quite happy with it ;) Thanks for listening!

**Comment by Lywx** [comment]
@mberasategi For some unknown reason, the AcademicMarkdown doesn't show Critic syntax highlighting in my sublime setup (as I remember it used to work). But I guess the way to make it work with Critic is similar to my previous post. Thanks for telling me your solution.
